misc.go

```diff
@@ -133,6 +133,12 @@ func init() {
 	initTokenFunc("'\"", startString)
 }
 
+// CheckInTokenMap indicates whether the target string is contained in tokenMap.
+func CheckInTokenMap(target string) bool {
```

Comment:
Is it necessary to expose a new function, will it be used outside the package?

Reply:
I forgot to unexport it after trying to use it in `FuncCallExpr.Restore()`. Updated.